src/Octoshift/Commands/ReclaimMannequin/ReclaimMannequinCommandArgs.cs

```diff
@@ -20,5 +21,10 @@ public override void Validate(OctoLogger log)
         {
             throw new OctoshiftCliException($"Either --csv or --mannequin-user and --target-user must be specified");
         }
+
+        if (string.IsNullOrEmpty(Csv) && SkipInvitation)
+        {
+            throw new OctoshiftCliException($"--csv must be specified to skip reclaimation email");
```

Comment:
Just a suggestion on how we can make this error message a little better:

```suggestion
            throw new OctoshiftCliException($"--csv must be specified when using the `--skip-invitation` option");
```

Reply:
Updated verbiage in c939758efc03a84ff993b188194864aea7966231